Added a markdown file containing information on the getAvailableLocal… (#3024)
Codecademy/docs: content/java/concepts/calendar/terms/getAvailableLocales/getAvailableLocales.md

---
Title: '.getAvailableLocales()'

> Description: 'Returns an array of all locales available in java runtime environment.'

Reviewer: ```suggestion
Description: 'Returns an array of all locales available in the Java runtime environment.'
```

Subjects:
  - 'Code Foundations'
  - 'Computer Science'
Tags:
  - 'Calendar'
  - 'Methods'
CatalogContent:
  - 'learn-java'
  - 'paths/computer-science'
---

> The `.getAvailableLocales()` method of the `Calendar` class is used to return an array of all locales. getAvailableLocals() method is a static method.

Reviewer: ```suggestion
The **`.getAvailableLocales()`** method of the `Calendar` class is used to return an array of all locales. In other words, an array of all locales for which there are `Calendar` instances.
```

## Syntax

```pseudo

> 
> public static Locale[] getAvailableLocales()  
> 

Reviewer: ```suggestion
Locale.getAvailableLocales()  
```

```

## Example

The following example displays the available locales.

```java

import java.util.Locale;  

public class JavaCalendargetavailablelocalsExample1 {  
public static void main(String args[]) {  

>        // an object of local class called strlocal(array) is created   

Reviewer: ```suggestion
       // An object of local class called strlocal(array) is created   
```

Reviewer: Comments should begin with a capital and all indentations should use 2 spaces not tabs

       Locale[] strlocal = new Locale[5];  
  
      // all available locals are accessed and stored in strlocal  
      strlocal = Locale.getAvailableLocales();  
  
      // print all available locals   
      System.out.println("The first 5 locales installed in runtime environment are :-\n");  
      for (int i = 0; i < 6; i++) {  
         System.out.println(strlocal[i].getISO3Country());  
        }  
   }  
}  

```

> This will output the following:

Reviewer: ```suggestion
This will output a result similar to the following:
```
